lib/apiservers/engine/backends/container.go

```diff
@@ -481,8 +481,18 @@ func (c *Container) ContainerRm(name string, config *types.ContainerRmConfig) er
 
 	// Use the force and stop the container first
 	secs := 0
+
 	if config.ForceRemove {
 		c.containerProxy.Stop(vc, name, &secs, true)
+	} else {
+		state, err := c.containerProxy.State(vc)
+		if err != nil {
+			return err
+		}
+		// force stop if container is broken to make sure container is deletable later
+		if state.Status == ContainerError {
```

Comment:
Ok, I just got broken was an actual status.  It's actually a description, meaning the status is error.

Can you change the comment to say "if container is in an error state"?  Or is broken the best description?